veg2hab/vegetatietypen.py

```diff
@@ -411,3 +412,194 @@ def opschonen_series(series: pd.Series) -> pd.Series:
         series = series.apply(lambda x: None if pd.isnull(x) or x == "" else x)
 
         return series
+
+
+class rVvN(BaseModel):
+    """
+    Format van VvN codes:
+    ## is cijfer ('1', '5', '10', '32', niet '01' of '04'), x is letter ('a', 'b', 'c' etc)
+    Normale VvN: ##xx##x, zoals 42aa1e
+    Behalve klasse is elke taxonomiegroep optioneel, zolang de meer specifieke ook
+    afwezig zijn (klasse-orde-verbond is valid, klasse-verbond-associatie niet)
+    Rompgemeenschappeen: ## rg ##, zoals 37rg2
+    Derivaatgemeenschappen: ## dg ##, zoals 42dg2
+    """
```

Comment:
Nu we 3 vegetatietype klassen hebben zou er best een superclass voor kunnen komen, maar veel meer nut dan "het is wel netjes" kon ik eigenlijk niet verzinnen, dus heb er geen tijd in gestoken. Veel functionaliteit is toch net subtiel anders, dus echt heel veel minder code krijg je er niet van denk ik. Mee eens?

Reply:
Het maakt vooral de typing wat netter, Union[SBB, rVvN, VvN] wordt dan gewoon VegtypeBase. 

Al is het best vaak zo, dat we alleen SBB en VvN supporten en bijvoorbeeld niet rVvN.